zavod/setup.py

```diff
@@ -46,6 +46,7 @@
         "requests_oauthlib",
         "sqlalchemy[mypy]",
         "structlog",
+        "transliterate",
```

Comment:
Let's please not include another transliteration mechanism - see other comments.

Reply:
Understood!